src/declarations/radio-buttons-react-native.d.ts

```diff
@@ -0,0 +1,40 @@
+declare module 'radio-buttons-react-native' {
```

Comment:
Is deze file voor de typescript types?

Reply:
Ja, die radio-buttons heeft geen types beschikbaar om te installeren :/